README.md

```diff
@@ -5,6 +5,7 @@ Guides for getting things done, programming well, and programming in style.
 
 * [Best Practices](/best-practices)
 * [Code Review](/code-review)
+* [How to](/howto)
```

Comment:
`howto` -> `how-to`

Also, I think it might be useful to briefly describe this page here. It's unclear as to what we can find by following this link.